cli/config/compose/services/vsphere.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  vsphere:
+    image: "nimmis/vcsim:${vsphereTag}"
```

Comment:
Nice! I didn't know this one. I guess this simulates a vsphere sever?

Reply:
Yeah, we think that for PRs it gives enough confidence that the code is working. That does not mean that we shouldn't start up a real vSphere environment during release preparation, which is something @v1v has been doing with @sayden